CHANGELOG.md

```diff
@@ -1,4 +1,89 @@
-## 4.2.4 (2024-04-02)
+## 4.2.4_bp2 (2024-05-14)
+
+### Bug fixes
+
+Data source
+
+- Unable to connect to ODP-Sharding data source [#2339](https://github.com/oceanbase/odc/pull/2339)
+
+User Management
+
+- Unable to delete OWNER or DBA user of archived project [#2359](https://github.com/oceanbase/odc/pull/2359)
+
+Shadow table synchronization
+
+- Syntax errors may occur in statements generated when a column has a default value [#2388](https://github.com/oceanbase/odc/pull/2388)
+
+Data Security
+
+- When there is an invalid database with the same name, desensitization fails [#2385](https://github.com/oceanbase/odc/pull/2385)
+
+Command line window
+
+- When entering a long SQL, the statement cannot be fully echoed [#2353](https://github.com/oceanbase/odc/pull/2353)
+
+Database archiving/cleaning
+
+- The status may be incorrectly set to success when a task is terminated [#2340](https://github.com/oceanbase/odc/pull/2340)
+- There is no problem with OceanBase MySQL archiving custom table name reports [#2313](https://github.com/oceanbase/odc/pull/2313)
```

Comment:
If the table names are different between the source and target when configuring data archiving tasks, an error occurs stating that the table does not exist.

Reply:
done